### ISSUE TITLE
Creating animations

### ISSUE THREAD

**Issue by shoaibmirza** [2dom/PixelTimes#11]
Hi, I'm trying to create my own animation but couldn't understand your instruction, can you please make it a little more clear? like step by step instruction. 
here is what I did. 
I download a few gif images and I put them into the same directory where is my image_to_array.sh and creat_program.py script is. then I tried to run image_to_array.sh from the terminal (Command Prompt) but it doesn't work. 
![Screenshot (89)](https://user-images.githubusercontent.com/17109348/70034402-5073aa00-15db-11ea-9fea-15906f102a81.png)

**Comment by 2dom** [comment]
Well..you have to install ffmpeg

**Comment by shoaibmirza** [comment]
also, how can I use image_to_array.sh for PNG or JEG images?

**Comment by shoaibmirza** [comment]
> Well..you have to install ffmpeg

Yes, it worked after installing FFmpeg. thanks :)  can you include that part in your instruction?
